src/reason-parser/jbuild

```diff
@@ -2,6 +2,12 @@
 
 (ocamllex (reason_lexer))
 
+; Use select.exe from janestreet/ppx_ast to choose lexer_warning.ml based on ocaml version
+(rule
+ ((targets (lexer_warning.ml))
+   (deps (../generate/select.exe lexer_warning.ml-4.06 lexer_warning.ml-default))
+   (action (bash "../generate/select.exe lexer_warning.ml-4.06 lexer_warning.ml-default > ${@}"))))
```

Comment:
@rgrinberg Do we need to prefix this with `bash` here?

Reply:
Since it's doing shell redirection, it's necessary. But the the use of bash itself is not justified. This can be with the `with-stdoutout-to` stanza instead:

```
(with-stdout-to ${@} (run ../generate/select.exe lexer_warning.ml-4.06 lexer_warning.ml-default))
```

Reply:
That's what I'm looking for! Thanks @rgrinberg!
@ncihnegn Care to try that and then we can merge?